tests/functional/shutdown/12-bad-port-file-check/suite.rc

```diff
@@ -20,7 +20,7 @@
         script = """
 wait "${CYLC_TASK_MESSAGE_STARTED_PID}" 2>/dev/null || true
 # Corrupt port file and don't report back to suite
-SRVD="$(cylc get-global-config --print-run-dir)/${CYLC_SUITE_NAME}/.service"
+SRVD="$RUN_DIR/${CYLC_SUITE_NAME}/.service"
 echo 'Haha! I have corrupted the port file!' >"${SRVD}/contact"
```

Comment:
I see you've touched a small handful of tests that have been repurposed to test our handling of the contact file, rather than the (long gone) "port file" - would you mind quickly renaming and re commenting these for clarity? (It's off-topic and not the fault of this PR, but quick and easy to fix now before forgetting it again...)

Reply:
(meh, actually don't bother ... doesn't really matter).